### DISCUSSION TITLE
looking for program support

### DISCUSSION THREAD

**Discussion by festation2003** [File-New-Project/EarTrumpet#1585] [Ideas]
G'day, just wondering if its possible to get the program KDE Connect added so its supported, It does not show up in the list of "running apps" even if the main screen is up so i cannot make it loader then other programs

**Comment by riverar** [comment]
@festation2003 Hmm. I'll take a look. Are you using the installer or? (kdeconnect-kde-release_24.02-3692-windows-cl-msvc2022-x86_64)

**Reply by festation2003** [reply]
Version 24.02.1, pretty sure then installer as usually use direct sites for programs like that over windows store

**Reply by riverar** [reply]
Thanks, can you give me a brief rundown on how you use this app? With phone audio maybe? Or?

**Reply by festation2003** [reply]
The main thing is purely notifications, handling messaging services, 2fa notifications ect, hence the controlling of volumes would be handy for it

**Comment by festation2003** [comment]
It will be the installer I think, I don't usually use the windows store for things like that

Yahoo Mail: Search, organise, conquer 
 
  On Mon, 5 Aug 2024 at 2:50 am, Rafael ***@***.***> wrote:   


@festation2003 Hmm. I'll take a look. Are you using the installer or? (kdeconnect-kde-release_24.02-3692-windows-cl-msvc2022-x86_64)
